domains/relicrb.json

```diff
@@ -0,0 +1,11 @@
+{
+    "description": "Relic's Personal Site",
+    "repo": "https://github.com/RelicRB/RelicRB.github.io",
+    "owner": {
+        "username": "RelicRB",
+        "email": "relicrb@gmail.com",
+    },
+    "record": {
+        "CNAME": "RelicRB.github.io"

```

Comment:
404?

Reply:
Hello!,
          I fixed it now it is showing my website will you please, try it again!!

Reply:
> 404?
i fixed it. Can you try it again?